### ISSUE TITLE
The acquireTokenSilent will always try to renew a new token instead of obtaining from the locale cache

### ISSUE THREAD

**Issue by kafka-yu** [AzureAD/microsoft-authentication-library-for-js#2456]
>> Please follow the issue template below. Failure to do so will result in a delay in answering your question.

## Library
- [x] `msal@1.4.1` 
- [ ] `@azure/msal-browser@2.x.x`
- [ ] `@azure/msal-angular@0.x.x`
- [ ] `@azure/msal-angular@1.x.x`
- [ ] `@azure/msal-angularjs@1.x.x`

**Important**: Please fill in your exact version number above, e.g. `msal@1.1.3`.

## Framework
React
## Description
1. I first time logged in, and get the access token successfully by acquireTokenSilent method.
2. Next, I found it always sends a request to STS to get a new token when the previous one is not expired.

## Error Message
Below is the logs
![image](https://user-images.githubusercontent.com/3942409/96572862-f4ec0100-12ff-11eb-8043-ce76ce188bf0.png)

## Security
- [ ] Is this issue security related?

## Regression
- [ ] Did this behavior work before? 
Version:

## MSAL Configuration

```js
// Provide configuration values here.
// For Azure B2C issues, please include your policies.
```
{
   forceRefresh: false,
   scopes: [ 'User.Read', 'User.Read.All']
}
```

## Reproduction steps
```js
// Provide relevant code snippets here.
// For Azure B2C issues, please include your policies.
```
1. First call `userAgentApplication.loginPopup()` with the above parameter.
2. Second time call `userAgentApplication.acquireTokenSilent()` with the above parameter, to check if it gets token successfully.
3. Call `userAgentApplication.acquireTokenSilent()` with the above parameter again to check if it gets the cached token
## Expected behavior
It should not send request to STS to get a new token when the previous cached one is not expired.

## Browsers/Environment
- [ ] Chrome
- [ ] Firefox
- [x] Edge
- [ ] Safari
- [ ] IE
- [ ] Other (Please add browser name here)

**Comment by GraemeF** [comment]
I noticed this too, @azure/msal-browser@2.2.0

**Comment by PeterBowater** [comment]
Also noticed here, have had to fall back onto popups. Not really ideal.

**Comment by kafka-yu** [comment]
> Also noticed here, have had to fall back onto popups. Not really ideal.

I've fallbacked to 1.3.4, which works fine...

**Comment by hectormmg** [comment]
@kafka-yu closing since this is a known issue in `msal@1.4.x` that is being addressed in #2376 and should be fixed in the 1.4.2 release. You can track progress in the original issue #2370. Thanks!

**Comment by kafka-yu** [comment]
@hectormmg thanks
